build/packaging/linux/conf/kend.conf

```diff
@@ -56,3 +56,32 @@ ADDITIONAL=""
 
 DATA_DIR=
 LOG_DIR=$DATA_DIR/logs
```

Comment:
@whoisxx Instead of adding `start-docker`, how about using stdout if `LOG_DIR` is not set?

Reply:
@KimKyungup It could be done either way. is there any advantage to do so?

Reply:
My idea is more complex. Please forget it. :)